src/components/card/EmissionsCard.tsx

```diff
@@ -0,0 +1,78 @@
+import type { FC } from 'react'
+import {
+  TruckIcon,
+  TrashIcon,
+  BoltIcon,
+  GlobeAmericasIcon,
+} from '@heroicons/react/24/outline'
+import clsx from 'clsx'
+
+export interface CardProps {
+  title: string
+  metric: string
+  description: string
+  color: string
+  type: string
+}
+
+interface IconProps {
+  type: string
+}
+
+const Icon: FC<IconProps> = ({ type }) => {
+  let icon
+
+  switch (type) {
+    case 'gasoline':
+      icon = <TruckIcon />
+      break
+    case 'trashbag':
+      icon = <TrashIcon />
+      break
+    case 'home':
+      icon = <BoltIcon />
+      break
+    case 'tree':
+      icon = <GlobeAmericasIcon />
+      break
+    default:
+      icon = <GlobeAmericasIcon />
+  }
+
+  return <div className="w-8"> {icon}</div>
+}
+
+const EmissionsCard: FC<CardProps> = ({
+  title,
+  metric,
+  description,
+  color,
+  type,
+}) => {
+  return (
+    <div className="p-8">
+      <h2 className="p-2 font-bold">{title}:</h2>
```

Comment:
```suggestion
      <h2 className="p-2 subheader-1">{title}:</h2>
```

The Figma uses graph-ff-condensed font here, which is in the repo as `subheader-1` (this is a new change since you've last been here :smile: )